src/test/java/org/openmrs/contrib/qaframework/automation/FindPatientSteps.java

```diff
@@ -40,9 +40,9 @@ public void noPatients() {
 		assertNotNull(getElement(By.className("dataTables_empty")));
 	}
 
-	@And("User enters John Smith")
-	public void enterJohnSmith() {
-		findPatientPage.enterPatient("John Smith");
+	@And("User enters {string} patientName")
+	public void enterPatientName(String patientName) {
```

Comment:
@sherrif10 why not giving a generic methodName to cater for searching a patient using patientIdentifier or patIentName ?

Reply:
That is already catered for and perhaps in steps class , this is was to improve the logical way of improving code using cucumber table.  There are number of different ways where this has been done , especially in the[ login. feature](https://github.com/openmrs/openmrs-contrib-qaframework/blob/master/src/test/resources/features/refapp-2.x/02-login/login.feature) as an example.